### PR TITLE
🚀Pro Manifest Improvements

### PR DESCRIPTION
**Pull request by wss-rbrennan**
Found that basic configurations are needed when deploying to non local cluster.

This PR adds the following

- cleanupInfrastructure.ps1 
  a script to quickly delete installed infrastructure components, 
- Add zookeeper-zoo-configmap.yaml
  a basic zoo.cfg file for single instance operation
- init containers to configure kafka and zookeeper
- setting security context to user 1001 to match bitnami containers
- set default resource requests and limits for all pods.  These are arbitrary and should be set to the users specific needs.

### CODE REVIEW

**Comment by cosmos-explorer** [comment]
Hello @wss-rbrennan, when I test the code, I see this error, it cannot start the kafka pod, it may be related to this change:
`setting security context to user 1001 to match bitnami containers`
image

File: kubernetes/pro/application/ui-service.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ kind: Service
 metadata:
   labels:
     app: ui
-  name: ui
+  name: standard-ui
```
Comment (cosmos-explorer) [inline kubernetes/pro/application/ui-service.yaml]: Why this change?

Comment (wss-rbrennan) [reply kubernetes/pro/application/ui-service.yaml]: ui change was a mistake, fixed with 36a4f4f, security context fixed with 8688832

**Review by cosmos-explorer — APPROVED** [review]

**Comment by cosmos-explorer** [comment]
LGTM, thank you
